src/wazuh_modules/vulnerability_scanner/src/vulnerabilityScannerFacade.cpp

```diff
@@ -18,6 +18,12 @@
 
 constexpr auto VULNERABILITY_SCANNER_TEMPLATE = "queue/indexer/vd_states_template.json";
 constexpr auto DEFAULT_QUEUE_PATH = "queue/sockets/queue";
+constexpr auto DELTA_EVENTS_QUEUE_PATH = "queue/vd/deltas";
+constexpr auto SYNCHRONIZATION_EVENTS_QUEUE_PATH = "queue/vd/sync";
+constexpr auto VD_STATE_QUEUE_PATH = "queue/vd/state_track";
```

Comment:
We must make sure that this DB isn't distributed with the initial content.

Reply:
It shouldn´t. We can change the path to make sure about it

Reply:
The current workflow would do it

https://github.com/wazuh/wazuh/blob/62a5a6752bfcf934a7c7bdf3609f89d5bea0f462/.github/workflows/vulnerability-scanner-generate-database.yml#L80-L98

Reply:
Ok great, I'll move the path and avoid change in the action. Thanks!

Reply:
Done!